doc/react/README.md

```diff
@@ -0,0 +1,146 @@
+# @hatchifyjs/react
```

Comment:
Yeah, Nils will be doing a bunch of restructuring of the docs, so stuff will be moving. My intention with using "@hatchifyjs/react" instead of just "react" was simply because it looked weird as hell to me to just have it be "react" (because of react). I'll leave that up to y'all to decide.